client/src/components/Header.js

```diff
@@ -8,14 +8,14 @@ import Logo from "./logonobg.png";
 
 const Header = () => {
   return (
-    <div class="ui menu inverted">
-      <div class="header item">
+    <div className="ui menu inverted">
```

Comment:
So should we be using className instead of class? Just for reference. I've been using class in my own PR.

Reply:
Yeah, it got rid of all the errors in my terminal. I updated all of them to use className now